domino/domino.py

```diff
@@ -36,6 +36,14 @@ def __init__(self, project, api_key=None, host=None, domino_token_file=None):
             self._logger.error(error_message)
             raise Exception(error_message)
 
+    @property
+    def log(self):
```

Comment:
@abhijeet2096 I'm being defensive here because I see a `staticmethod` on the class and you may add more, but realistically this should never not be initialized.